tutorials/independence.py

```diff
@@ -74,6 +74,7 @@
 # .. note::
 #
 #    :Pros: - Very accurate in certain settings
+#           - Has fast implementation
 #    :Cons: - Very slow (computationally complex)
 #           - Test statistic not very interpretable, not between (-1, 1)
 #
```

Comment:
Give a toy example using your fast implementation. You can use the `Dcorr section` as an example for the `auto` stuff

Reply:
Toy example made!